std_msgs/Header header
string tracker_name
MocapObject[] tracked_objects
